docs/snippets/common/button-story-argtypes-with-subcategories.js.mdx

```diff
@@ -0,0 +1,48 @@
+```js
+// Button.stories.js
+
+export default {
+  title: 'Button',
+  component: Button,
+  argTypes: {
+    // assigns the argTypes to the Colors category
+    backgroundColor: {
+      control: 'color',
+      table: {
+        category: 'Colors',
+        // assigns the argTypes to a specific subcategory
+        subcategory: 'Button colors'
+      }
+    },
+    primary: {
+      table: {
+        category: 'Colors',
+        // assigns the argTypes to a specific subcategory
+        subcategory: 'Button style'
+      }
+    },
+    // assigns the argType to the Text category
+    label: {
+      table: {
+        category: 'Text',
+        // assigns the argTypes to a specific subcategory
+        subcategory: 'Button contents'
+      }
+    },
+    // assigns the argType to the Events category
+    onClick: {
+      table: {
+        category: 'Events',
+        // assigns the argTypes to a specific subcategory
```

Comment:
Is this comment useful?

Reply:
@shilman just updated the snippet with a change. Just left one item commented out for clarity purposes. The others are cleared.